anago

```diff
@@ -1284,6 +1284,9 @@ archive_release () {
     text="contents"
   fi
 
+  # Remove temporary password file so not to alarm passers-by.
+  logrun find $WORKDIR -type f -name rsyncd.password -exec rm {};
```

Comment:
can you limit the find to specific directories? instead of searching

Reply:
I could but why not be broad about removing such files?